po/es.po

```diff
@@ -2122,27 +2142,27 @@ msgstr "Dependencias"
 #: modules/computer/os.c:36 modules/computer/os.c:37 modules/computer/os.c:38
 #: modules/computer/os.c:39
 msgid "GNU C Library"
-msgstr "Libreria C GNU"
+msgstr "Biblioteca C de GNU"
```

Comment:
must be Libreria, there's not books here!

Reply:
"Librería" is commonly used but it is a literal (and incorrect) translation for "library". "Biblioteca" **is** the right translation. We are not talking about books here but about a collection of data.

In common language (not IT related), "librería" is a place to buy books and "biblioteca" is a place where you read and can rent books, but (most of the times) you can't buy books there. So there're books in both indeed. :)